server/wavelength/modules/economy/index.js

```diff
@@ -0,0 +1,124 @@
+// Used with nef
+
+'use strict';
+
+const FS = require(('../../../../.lib-dist/fs')).FS;
+const DB_PATH = 'config/database'; // path to your database folder
+
+module.exports = class Economy {
+	constructor(name, dbName) {
+		this.name = name;
+		this.dbName = dbName;
+		this.db = Db[dbName];
+
+		this.transactionLogs = FS('logs/transactions.txt');
+		this.tLogs = this.transactionLogs.createAppendStream();
+	}
+
+	grammarizeName(value) {
+		return value !== 1 ? `${this.name}s` : this.name;
```

Comment:
This can simply be written as:
```js
grammarizeName(value) {
   return `${this.name}${Chat.plural(value)}`;
}
```
I'm also not the biggest fan of the name - maybe `getCurrencyLabel` or something as that that's more descriptive and clear?

Reply:
Also, since you have written `${value} ${econWL.grammarizeName(value)}` several times across the plugin file, I would recommend this definition instead:
```js
getCurrencyLabel(value, includeValue = true) {
   let buf = '';
   if (includeValue) buf += `${value} `;
   buf += `${this.name}${Chat.plural(value)}`;
   return buf;
}
```